powers.c

```diff
@@ -0,0 +1,26 @@
+/* This program accepts a user inputted number and prints all powers of 2 so that 2^n is less than that number. - Amanda Cuevas */
+#include <stdio.h>
```

Comment:
you misunderstood the assignment. n isn't the number being inputted, it's the number that 2^n should be less than. So if you input 10, it should print 2^0 = 1, 2^1 = 2, 2^2 = 4, 2^3 = 8.